app/Livewire/Project/New/GithubPrivateRepositoryDeployKey.php

```
@@ -199,6 +234,21 @@ public function submit()
+            if ($this->envImported && count($this->envExampleVars) > 0) {
+                DB::transaction(function () use ($application): void {
+                    foreach ($this->envExampleVars as $key => $value) {
+                        EnvironmentVariable::create([
+                            'key' => $key,
+                            'value' => $value,
+                            'resourceable_type' => $application->getMorphClass(),
+                            'resourceable_id' => $application->id,
+                            'is_preview' => false,
+                        ]);
+                    }
+                });
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'submit'. (undefined)

(StaticAccess)